vignettes/event_process_mining.Rmd

```diff
@@ -0,0 +1,70 @@
+---
+title: "Event Process Mining"
+output: 
+  html_document:
+    toc: true
+    number_sections: true
+vignette: >
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteIndexEntry{Event Process Mining}
+  %\VignetteEncoding{UTF-8}
+---
+
+# Introduction
+
+Kaiaulu interface to Process Mining for Github events relies on [pm4py](https://github.com/process-intelligence-solutions/pm4py). This vignette demonstrates how to use Kaiaulu to interact with GitHub event data visually using pm4py for further analysis.
+
+## Requirements
+
+-   Gihub Access Token
+-   Python (for visualizations)
+-   Reticulate package (to interface with python)
+-   Python environment with [pm4py](https://github.com/process-intelligence-solutions/pm4py) installed
+
+### Install Reticulate
+
+```{r}
+install.packages("reticulate")
```

Comment:
You should not use reticulate. This is a strong dependency on Kaiaulu side. Rather, it is the Python script that should call this exec script.